### DISCUSSION TITLE
How to pair LYWSD03MMC with Smart Life or Tuya

### DISCUSSION THREAD

**Discussion by eduedp** [pvvx/ATC_MiThermometer#382] [Q&A]
I flashed Custom firmware v4.3 and I'm trying to connect it to Smart Life or Tuya, but it's not finding the device.
Is it possible to connect to those apps without a gateway ?

Thanks.

**Comment by evlo** [comment]
Not possible even **with** the tuya bluetooth gateway

**Comment by devbis** [comment]
It is reported that Zigbee firmware for 03mmc may work with (some?) Tuya zigbee gateways/hubs.
